static/src/javascripts/projects/commercial/modules/third-party-tags.js

```diff
@@ -59,17 +58,21 @@ const insertScripts = (
     advertisingServices: Array<ThirdPartyTag>,
     performanceServices: Array<ThirdPartyTag>
 ): void => {
-    oldCmp.onGuConsentNotification('performance', state => {
-        if (state) {
+    onConsentChange(state => {
+        // TODO: decide when to load performance scripts
+        if (state.ccpa && !state.ccpa.doNotSell) {
+            addScripts(performanceServices);
+        } else if (
+            state.tcfv2 &&
+            state.tcfv2.consents[7] && // Measure ad performance
+            state.tcfv2.consents[8] && // Measure content performance
+            state.tcfv2.consents[10] // Develop and improve products
+        ) {
```

Comment:
We need to have guidance on this. It might be better to do this in a subsequent PR and always load performance scripts for the time being…

Reply:
Following a discussion with @ripecosta, the dummy `oldCmp.onGuConsentNotification` was always returning true. For now, we'll only run performance scripts if all consents are given.